### ISSUE TITLE
If I run to_dict, the protobuf object is changed to dict type

### ISSUE THREAD

**Issue by wildfire810** [danielgtaylor/python-betterproto#158]
programDict = program.to_dict(betterproto.Casing.CAMEL, True)

before:
![image](https://user-images.githubusercontent.com/17704611/96239085-d620fe00-0fd1-11eb-8157-ed5cd552d4f0.png)

after:
![image](https://user-images.githubusercontent.com/17704611/96239185-fa7cda80-0fd1-11eb-8d9d-61b1dfa9f035.png)

**Comment by wildfire810** [comment]
maybe it is object in dict, will be converted to dict in dict

**Comment by upcFrost** [comment]
yep, https://github.com/danielgtaylor/python-betterproto/issues/151
